src/read.c

```diff
@@ -265,55 +267,55 @@ GAP_STATIC_ASSERT(sizeof(LHSRef) <= 8, "LHSRef is too big");
 /****************************************************************************
 **
 */
-static UInt EvalRef(const LHSRef ref, Int needExpr)
+static UInt EvalRef(ReaderState * rs, const LHSRef ref, Int needExpr)
```

Comment:
`EvalRef` now needs the `ReaderState * rs` so that it can extract the `IntrState` from it and pass that on.